shared/xccdf/system/auditing.xml

```diff
@@ -848,14 +814,12 @@ nefarious activities in log files, as well as to confuse network services that
 are highly dependent upon an accurate system time (such as sshd). All changes
 to the system time should be audited.</rationale>
 <platform idref="cpe:/a:machine" />
-<ident prodtype="rhel7" cce="27290-6" />
+<ident cce="27290-6" prodtype="rhel7" />
```

Comment:
I assume all reordering of attributes was done by the script.

Reply:
yes, no way I am doing this kind of work manually :-P